src/tools/kdb/CMakeLists.txt

```diff
@@ -21,7 +21,16 @@ if (BUILD_SHARED)
 	add_executable (kdb $<TARGET_OBJECTS:kdb-objects>)
 	add_dependencies (kdb kdb_gen_templates_generated)
 
-	target_link_libraries (kdb elektra-core elektra-kdb elektratools elektra-opts elektra-merge)
+	set (NOT_INCLUDED "")
+	plugin_check_if_included ("jni")
```

Comment:
I don't think this is worth the hassle: there are also other plugins which might needs pthreads.

Btw. on my system, `kdb` already links with `pthread` (checked with `ldd =kdb`).

Reply:
for me it wasn't. updated

Reply:
> Btw. on my system, `kdb` already links with `pthread` (checked with `ldd =kdb`).

Doesn't `ldd` also show libraries loaded at runtime via `dlopen`? Maybe something pulls in `pthread` dynamically, but it's not actually linked in the binary?